CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing
+
+Contributions are welcome. For all non-trivial changes, please create an issue first to help us track what is being worked on. PR titles should follow the angular style for compatibility with our automated release tool. See [here](https://www.conventionalcommits.org/en/v1.0.0/) for an explanation of the allowable commit message format.
+
+Breaking changes will cause the major version number of the library to be incremented. 
+`feat` changes will cause the minor version number to be incremented.
+`fix` and `perf` changes will cause the patch version number to be incremented.
+The commit messages for the foregoing will be automatically added to the changelog upon release.
```

Comment:
So, commit messages on a branch are just added as-is to the changelog?

Reply:
Correct